election-api-java/src/main/java/bbc/news/elections/model/ConstituencyResult.java

```diff
@@ -30,4 +18,20 @@ public ConstituencyResult(
     public Integer getSeqNo() { return seqNo; }
 
     public List<PartyResult> getPartyResults() { return partyResults; }
+
+    public void setId(Integer id) {
+        this.id = id;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public void setSeqNo(Integer seqNo) {
+        this.seqNo = seqNo;
+    }
+
+    public void setPartyResults(List<PartyResult> partyResults) {
+        this.partyResults = partyResults;
+    }
 }
```

Comment:
Minor and not your change, but I find this sort of thing a little embarrassing in something that's presented to the public.